qemu/tests/win_heavyload.py

```diff
@@ -58,6 +58,19 @@ def add_option(cmd, key, val):
             cmd += " /%s %s " % (key, val)
         return cmd
 
+    def cleanup_temp(session):
+        """
+        del temp file that created by heavyload in VM
+
+        :param session: VM session
+        """
+        check_temp_cmd = 'dir %temp% %tmp% /s /b | findstr jam*'
```

Comment:
Do you want to find temp file with "ja" , "jam" "jamm" .. and delete them ? 
If just want to find file with "jam" ,pls delete *.

Reply:
No, we can not, the file name is jam_*****,  it is  generated randomly by heavyload, thanks.

Reply:
So, check_temp_cmd = 'dir %temp% %tmp% /s /b | findstr jam'   is ok.

Reply:
Thanks,  updated with more strict pattern.

Reply:
Thanks,  updated with more strict pattern.